gxa/src/main/webapp/resources/css/popup-feedback.css

```diff
@@ -0,0 +1,200 @@
+.mm-popup {
```

Comment:
If you take copy-paste someone else's code (even if it is just CSS), it is good practice to attribute the source. If nothing else, it will serve as guidance for future developers that might wonder where the file is from. 

Also, you should add a note in the [atlas-smileys-feedback-form](https://github.com/ebi-gene-expression-group/atlas-smileys-feedback-form) Readme file to explain how to style the component, and link to the CSS example. Otherwise, someone running the component might think it's broken if they don't have the Atlas CSS.